sdk/storage/Azure.Storage/Azure.Storage.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>12.1.0-preview.1</Version>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+
+    <PackageType>meta-package</PackageType>
```

Comment:
Could you explain this package's scenario? How did you chose this `PackageType` instead of the default?

FYI, this will likely break package installation in Visual Studio:

* https://github.com/NuGet/Home/wiki/Package-Type-%5BPacking%5D
* https://github.com/NuGet/NuGet.Client/blob/3803820961f4d61c06d07b179dab1d0439ec0d91/src/NuGet.Core/NuGet.PackageManagement/InstallationCompatibility.cs#L187-L192

I would reach out to `nugetdisc@microsoft.com` and @nkolev92 for more information.

Reply:
Just to add to what @loic-sharma said, meta packages historically do not have a special package type. They just are normal dependency packages with no assets and only dependencies.

Reply:
I will remove this metadata. I added as a first attempt to try and deal with NU5128, based on some stuff I read in https://github.com/nuget/home/issues/8583, however that didn't work and I ended up needing to disable NU5128 so I suspect I don't need this PackageType.